src/main/resources/biz_de.properties

```
@@ -938,6 +938,7 @@ TenantData.canAccessParent = Verwaltung des übergeordneten Mandanten
 TenantData.canAccessParent.help = Ermöglicht es Anwendern dieses Mandanten den übergeodneten Mandant zu verwalten.
 TenantData.externalLoginIntervalDays = Interval für Externen Login
 TenantData.externalLoginIntervalDays.help = Steuert nach wieviel Tagen ein Benutzer, bei dem "Externer Login erforderlich" aktiv ist, sich erneut über ein externes System authentifizieren muss.
+TenantData.image = Mandantenlogo
```

Analog zu `Firmenname` unterhalb
```suggestion
TenantData.image = Firmenlogo
```